skills/codex-review-partner/scripts/run-review.sh

```
@@ -99,168 +107,73 @@ os.replace(tmp,p); os.chmod(p,0o600)
-if (verified_pid,verified_pgid) != (pid,pgid): raise SystemExit
-try: os.killpg(pgid,signal.SIGKILL)
-except ProcessLookupError: pass
-PY
-WATCHDOG_PID=$!
 wait "$CODEX_PID"; CODEX_EXIT=$?
-kill "$WATCHDOG_PID" 2>/dev/null; wait "$WATCHDOG_PID" 2>/dev/null
 set -e
 
-TIMED_OUT=false
-if [[ -f "$PRIVATE_DIR/timed-out" ]]; then TIMED_OUT=true; fi
+read -r RESULT_CODE RESULT_SIGNAL TIMED_OUT RESULT_CLEANUP < <(python3 - "$SUPERVISOR_RESULT" "$CODEX_EXIT" <<'PY'
```

Shellcheck (0.11.0)

[warning] RESULT_CLEANUP appears unused. Verify use (or export if used externally).

(SC2034)